<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="TX717-4441WGD" name="testscripts.SF.RegressionTestScriptsOpp.TC_SF_ABAMERCSSRecalculateRoundOffRule" tests="1" failures="0" timestamp="16 Jun 2017 19:23:10 GMT" time="153.829" errors="1">
  <testcase name="runTC_SF_ABAMERCSSRecalculateRoundOffRule" time="153.829" classname="testscripts.SF.RegressionTestScriptsOpp.TC_SF_ABAMERCSSRecalculateRoundOffRule">
    <error type="com.cognizant.framework.FrameworkException" message="Error while capturing the screenshot">
      <![CDATA[com.cognizant.framework.FrameworkException: Error while capturing the screenshot
	at com.cognizant.framework.selenium.SeleniumReport.takeScreenshot(SeleniumReport.java:94)
	at com.cognizant.framework.Report.handleStepInvolvingScreenshot(Report.java:287)
	at com.cognizant.framework.Report.updateTestLog(Report.java:228)
	at com.cognizant.Craft.DriverScript.exceptionHandler(DriverScript.java:719)
	at com.cognizant.Craft.DriverScript.executeCRAFTTestIterations(DriverScript.java:594)
	at com.cognizant.Craft.DriverScript.executeCraft(DriverScript.java:157)
	at com.cognizant.Craft.DriverScript.driveTestExecution(DriverScript.java:150)
	at testscripts.SF.RegressionTestScriptsOpp.TC_SF_ABAMERCSSRecalculateRoundOffRule.runTC_SF_ABAMERCSSRecalculateRoundOffRule(TC_SF_ABAMERCSSRecalculateRoundOffRule.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- runTC_SF_ABAMERCSSRecalculateRoundOffRule -->
</testsuite> <!-- testscripts.SF.RegressionTestScriptsOpp.TC_SF_ABAMERCSSRecalculateRoundOffRule -->
